### DISCUSSION TITLE
Specifying distributed load / moment in input file

### DISCUSSION THREAD

**Discussion by andrey1887** [idaholab/moose#26077] [Q&A Modules: Solid mechanics] [answered]
Hello. I would like to be able specify a distributed load for beam element in input file. How it can be done? For a nodal load it looks:
```
[NodalKernels]
  [./force_y2]
    type = ConstantRate
    variable = disp_y
    boundary = right
    rate = 1.0e-4
  [../]
[]
```
But what i should do with distributed load?

**Comment by GiudGiud** [comment]
Hello

Well the `right` boundary could be a nodeset, and the load is then distributed on each node of the nodeset

Guillaume

**Reply by andrey1887** [reply]
Ok, thank you.

**Reply by andrey1887** [reply]
Hello, @GiudGiud. This is acceptable solution, but is there a possibility to assign a load directly to element, not to nodes? I mean, that in this case there is possibility to define one value of load independently of elements length unlike in nodes variant.

**Reply by GiudGiud** [reply, marked as answer]
You can use BodyForce kernels for volume forces
or DiracKernels for point forces inside elements

**Reply by andrey1887** [reply]
Thank you, @GiudGiud .
